src/components/Events/Fall_GM/Fall_GM.js

```diff
@@ -5,57 +5,196 @@ import Countdown from 'react-countdown';
 
 export default class Fall_GM extends React.Component {
```

Comment:
Should delete the underscore in the component name

Reply:
Also, given that there's no state, could we make this a functional component?